### PR TITLE
RadonMixed/RadonBytes refactor Part1

### LINKED ISSUE #905: Rename RadonBytes for RadonMixed

### LINKED ISSUE #906: Create RadonBytes type

### PR DESCRIPTION
**Pull request by lrubiorod**
Close #905 
Close #906

### CODE REVIEW

File: rad/src/operators/string.rs [hunk 1]
```diff
@@ -4,6 +4,7 @@ use std::{error::Error, str::FromStr};
 use json;
 use serde_cbor::value::{from_value, Value};
 
+use crate::types::bytes::RadonBytes;
```
Comment (tmpolaczyk) [inline rad/src/operators/string.rs]: fmt

File: rad/src/operators/mixed.rs [hunk 2]
```diff
@@ -1,32 +1,96 @@
...
+    let radon_mixed_error =
+        RadonMixed::from(Value::try_from(String::from("Hello world!")).unwrap());
     assert_eq!(
         &to_float(radon_mixed_error).unwrap_err().to_string(),
         "Failed to convert string to float with error message: invalid float literal"
     );
 }
+
+#[test]
+fn test_as_integer() {
```
Comment (tmpolaczyk) [inline rad/src/operators/mixed.rs]: Add tests for `as_map` and `as_array`?

Comment (lrubiorod) [reply rad/src/operators/mixed.rs]: RadonMixed is going to die so...

Comment (tmpolaczyk) [reply rad/src/operators/mixed.rs]: Ok

**Pull request merged**